libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -133,7 +220,17 @@ class AP_Airspeed
     bool healthy(uint8_t i) const {
         return state[i].healthy && (fabsf(param[i].offset) > 0 || state[i].use_zero_offset) && enabled(i);
     }
-    bool healthy(void) const { return healthy(primary); }
+    bool healthy(void) const {
+        if (blend_state.enable == BLEND_FULL) {
+            if (blend_state.active) {
+                return true;
```

Comment:
why not return blend_state.active like the other ones?

Reply:
Done